src/components/TwitterButton/index.js

```diff
@@ -46,8 +44,8 @@ class SignInTwitter extends Component {
       <Redirect to={ROUTES.HOME} />
     ) : (
       <button type="submit" className="facebook-btn" onClick={this.handleClick}>
-        <FacebookImg className="facebook-btn__img" />
-        <span className="facebook-btn__text">Facebook</span>
+        <TwitterImage className="facebook-btn__img" />
```

Comment:
Change the classes names 
```suggestion
        <TwitterImage className="twitter-btn__img" />
```